src/components/Content/Content.js

```diff
@@ -2,47 +2,65 @@ import './Content.scss'
 import TopBar from '../Topbar/Topbar'
 import Products from '../Products/Products';
 import Pagination from '../Pagination/Pagination';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
 
 function Content() {
-    const totalResult = 21469;
-    const currentPage = 1;
-    const totalPage = 7;
-    const products = [
-        {
-            name: "Sharp - 50 Class (49.5 Diag.) - LED - 1080p - Smart - HDTV Roku TV - Black",
-            rate: 0,
-            price: 999.99,
-            src: "https://cdn-demo.algolia.com/bestbuy-0118/3602009_sb.jpg"
-        },
-        {
-            name: "Sharp - 50 Class (49.5 Diag.) - LED - 1080p - Smart - HDTV Roku TV - Black",
-            rate: 5,
-            price: 999.99,
-            src: "https://cdn-demo.algolia.com/bestbuy-0118/3602009_sb.jpg"
-        },
-        {
-            name: "Sharp - 50 Class (49.5 Diag.) - LED - 1080p - Smart - HDTV Roku TV - Black",
-            rate: 2,
-            price: 999.99,
-            src: "https://cdn-demo.algolia.com/bestbuy-0118/3602009_sb.jpg"
-        },
-        {
-            name: "Sharp - 50 Class (49.5 Diag.) - LED - 1080p - Smart - HDTV Roku TV - Black",
-            rate: 1,
-            price: 999.99,
-            src: "https://cdn-demo.algolia.com/bestbuy-0118/3602009_sb.jpg"
-        }
-    ]
+    const initPaging = {
+        _page: 1,
+        _limit: 16
+    }
+    const [pagination, SetPagination] = useState(() => {
+        return initPaging
+    })
```

Comment:
```suggestion
    const [pagination, setPagination] = useState(initPaging)
```
Như này là được rồi nhé. Nhưng sao phải cần 1 cái pagination 1 cái params nhỉ
Dùng chung cả thứ cũng được mà
```
const { _page, _limit } = pagination
axios.get(url, {
   _page,
   _limit
})
```
Chứ tạo 2 cái phải update cả 2 đấy, nhọc lắm